extensions/protocols/dcp/presentation-api/src/main/java/org/eclipse/edc/identityhub/api/verifiablecredential/PresentationApiController.java

```diff
@@ -126,6 +150,19 @@ public Response queryPresentation(@PathParam("participantContextId") String part
         }
     }
 
+    private Result<DcpProtocol> parseProtocol(JsonObject query) {
```

Comment:
I think we'll need similar functionality in other DCP endpoints (issuance) as well. So maybe moving this into some centrally available class could be useful.

Reply:
ah that's true. ok, nvm then :)